pdr_backend/pdr_dashboard/test/filtered_volume_max_1400.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I don't think this should be a constant test file. When we test against it, we can use the the empty list directly.

Reply:
Let's keep it like this, in case the test result will be changed in the future, also keeping it consistent with other result formats